itests/org.openhab.binding.mqtt.ruuvigateway.tests/src/main/java/org/openhab/binding/mqtt/ruuvigateway/RuuviGatewayTest.java

```diff
@@ -132,6 +143,28 @@ public class RuuviGatewayTest extends MqttOSGiTest {
     private @NonNullByDefault({}) RuuviGatewayDiscoveryService ruuviDiscoveryService;
     private Set<Thing> things = new HashSet<>();
 
+    @BeforeEach
+    public void setup() {
+        when(mockedUnitProvider.getUnit(any())).then(i -> {
+            Class clazz = i.getArgument(0);
+            if (Temperature.class.equals(clazz)) {
+                return SIUnits.CELSIUS;
+            } else if (Acceleration.class.equals(clazz)) {
+                return Units.METRE_PER_SQUARE_SECOND;
+            } else if (Dimensionless.class.equals(clazz)) {
+                return Units.ONE;
+            } else if (ElectricPotential.class.equals(clazz)) {
+                return Units.VOLT;
+            } else if(Pressure.class.equals(clazz)) {
```

Comment:
Is Spotless disabled for tests? Just curious. I would have expected:
```suggestion
            } else if (Pressure.class.equals(clazz)) {
```